File: MiniJira.Server/Entities/Project.cs
namespace MiniJira.Server.Entities
{
    public class Project
    {
        public Guid Id { get; set; } = Guid.NewGuid();
        public string Key { get; set; } = null!;
        public string Name { get; set; } = null!;
        public string? Description { get; set; }
        public DateTime? CreatedAt { get; set; }
        public DateTime? UpdatedAt { get; set; }

        public Guid OwnerId { get; set; }
        public User? Owner { get; set; }
        public ICollection<ProjectMember> ProjectMembers { get; set; } = new List<ProjectMember>();
    }
}

File: minijira.client/src/models/Project.ts
import { Dayjs } from "dayjs";

export interface Project {
  id?: string;
  key?: string;
  name?: string;
  description?: string;
  ownerId?: string;
  ownerName?: string;
  createdAt?: Dayjs;
  updatedAt?: Dayjs;
  memberIds?: string[];
}

export interface ProjectFilter {
  pageIndex?: number;
  pageSize?: number;
  memberId?: string;
}